compiler/next/include/chpl/AST/ASTTag.h

```diff
@@ -87,7 +87,7 @@ enum ASTTag {
 #undef AST_LEAF
 #undef AST_BEGIN_SUBCLASSES
 #undef AST_END_SUBCLASSES
-#undef IS_AST
+#undef IS_BASE_CLASS_AST
```

Comment:
I was expecting these to be merged into a single `#define` grouping with only one `#include` of the classes header file.